services/website/website.tester.js

```diff
@@ -3,35 +3,37 @@
 const t = (module.exports = require('../tester').createServiceTester())
 
 t.create('status of http://shields.io')
-  .get('/website/http/shields.io.json?style=_shields_test')
-  .expectBadge({ label: 'website', message: 'online', color: 'brightgreen' })
+  .get('/http/shields.io.json?style=_shields_test')
+  .expectBadge({ label: 'website', message: 'up', color: 'brightgreen' })
```

Comment:
I changed the defaults here to match what we did for nodeping. They seem a bit nicer, though I don't feel strongly about it.

Reply:
Works for me